components/02-components/03-card/01-action-card/action-card.scss

```diff
@@ -9,63 +9,60 @@
 	.content-card-content .h5 {
 		font-size:28px;
 	}
-	
-	.content-card-list {
-		.card-flex-box:last-child {
-			padding-right: 0px;
-		}
-		
-		.content-card {
-			.content-card-img {
-				img {
-					width: 100%;
-					height: 210px;
-					-o-object-fit: cover;
-					object-fit: cover;
-				}
-			}
-
-			.content-card-content {
-				padding: 26px 33px 87px 33px;
-				position: relative;
-				border-top: 6px solid $orange;
 
-				p {
-					font-size: 14px;
-					color: $blue;
-				}
+	.content-card {
+		.content-card-img {
+			img {
+				width: 100%;
+				height: 210px;
+				-o-object-fit: cover;
+				object-fit: cover;
+			}
+		}
 
-				.content-card-link {
-					line-height: 1.1;
-					font-size: 25px;
-					margin-bottom: 24px;
-					color: $orange-a11y;
-					font-weight: 700;
-					display: block;
-					text-decoration: none;
-				}
+		.content-card-content {
+			padding: 26px 33px 87px 33px;
```

Comment:
Do you think it needs that much bottom padding? Should we do a min-height instead?

Reply:
hmm, that was already there, but yeah, 87px is a bit much, maybe an artifact copied over from the card with the right bottom clipping?